artemis-protocols/artemis-stomp-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/stomp/StompConnection.java

```diff
@@ -582,6 +582,27 @@ public void handleFrame(StompFrame request) {
       }
    }
 
+   public void logFrame(StompFrame request, boolean in) {
+      if (ActiveMQStompProtocolLogger.LOGGER.isDebugEnabled()) {
```

Comment:
for direct calls on debug and trace, I would an individualized Logger.


Logger logger = Logger.getLogger(StompConnection.class)


we should use Logger only for INFO messages with ID... as of debugging this, it gets fairly difficult to do it without the individual logs. Just wait till you ask debug loggers from users and it gets fairly difficult.

Reply:
Understood.  I'll change it.

Reply:
Done.